index.Rmd

```diff
@@ -51,7 +51,7 @@ Band4 <- ("B05.tif")
 Band5 <- ("B06.tif")
 Band6 <- ("B07.tif")
 Band7 <- ("B08.tif")
-
+ 
```

Comment:
Maybe mention why there's B and B .tif files? Unless I missed it, I'm not sure what these different files mean.